pkg/graphviz/traceflow.go

```diff
@@ -26,89 +26,53 @@ import (
 	opsv1alpha1 "github.com/vmware-tanzu/antrea/pkg/apis/ops/v1alpha1"
 )
 
+const (
+	darkRed    = "#B20000"
+	mistyRose  = "#EDD5D5"
+	fireBrick  = "#B22222"
+	ghostWhite = "#F8F8FF"
+	gainsboro  = "#DCDCDC"
+	lightGrey  = "#C8C8C8"
+	silver     = "#C0C0C0"
+	grey       = "#808080"
+	dimGrey    = "#696969"
+)
+
 var (
 	clusterSrcName = "cluster_source"
 	clusterDstName = "cluster_dest"
```

Comment:
cluster_destination seems better since the source is not an abbreviation.

Reply:
Variable clusterDstName changed.